path_generator:
  ros__parameters:
  #4 points stored as a flat list of doubles.
  # For each point: [x, y]
  path_points: [0.0, 0,2, 2.0, 2.0, 2.0, 0,0, 0.0, 0.0]
  square_side: 2.0
  robust_margin: 0.9